test-conf/tests/drivers/dma/cyclic/boards/tt_blackhole_tt_blackhole_smc.overlay

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (c) 2025 Tenstorrent AI ULC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	aliases {
+		dma0 = &dma1;
```

Comment:
This won't work because we'll have a conflict with the dma0 that is added in #657